front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -536,6 +574,23 @@ export function AgentMessage({
           </div>
         ) : null}
 
+        {streamedVisualizations.map(({ index, visualization }) => {
```

Comment:
This is def not final it's just to display something until we plug Aric's work to render the graph.